tests/test_target.py

```diff
@@ -37,23 +36,57 @@ def build_targets():
 
     targets = []
 
+    if target_configs.get('AndroidTarget') is not None:
+        print('> Android targets:')
```

Comment:
I'd still use a logger there. If you don't, any code that collects the log from logging but not stdout will miss this part. `pytest` has some integration with logging as it can set the log level, which can also be useful. If we really want to be lazy here, you can still use `logging.info` and that won't be much more to type than `print`

Reply:
Done.